developer_guide/installation/partners_portal_archive.rst

```diff
@@ -0,0 +1,34 @@
+Getting Akeneo Partners Portal access
+*************************************
+After obtaining your Enterprise Edition license you should have recieved an access to the Akeneo Partners Portal.
+
+If you haven't recieved it you can request one at: http://helpdesk.akeneo.com/
+
+If you are a developer on the project you should request an account to access the project to your company manager (supervisor).
+
+Getting Enterprise Edition access via the Partners Portal
+*********************************************************
+By default Composer (https://getcomposer.org/) uses packagist.org to retrieve all open source packages and their updates.
+To download the PIM Enterprise Edition you have to get access to our private enterprise edition repository by transmitting your SSH public key to our systems.
```

Comment:
I'd be more likely to use "to send" rather than "to transmit" as "to transmit" seems too telecommunication-related to me.